edge-apps/clock/static/css/styles.css

```diff
@@ -0,0 +1,918 @@
+    height: 1.79rem; /* 25px / 14 */
+    background: #fff;
+    border-radius: 1.79rem; /* 25px / 14 */
+    z-index: 999;
+}
+
+.hands_box .minute i {
+    width: 0.5rem; /* 7px / 14 */
+    height: 13.57rem; /* 190px / 14 */
+    background: #fff;
+    position: absolute;
+    box-shadow: 0px 0px 2.14rem 0px rgb(0 0 0 / 97%); /* 30px / 14 */
+}
+
+.hands_box .minute i::before {
+    content: "";
+    width: 1.86rem; /* 26px / 14 */
+    height: 14.71rem; /* 206px / 14 */
+    background: #ffffff;
+    position: absolute;
+    bottom: 1.79rem; /* 25px / 14 */
+    left: 50%;
+    transform: translateX(-50%);
+    border-radius: 1.79rem; /* 25px / 14 */
+    z-index: 1;
+    box-shadow: 0px 0px 1.71rem 0px rgb(0 0 0 / 26%); /* 24px / 14 */
+}
+
+.hands_box .minute i::after {
+    content: "";
+    width: 1.07rem; /* 15px / 14 */
+    height: 13.93rem; /* 195px / 14 */
+    background: var(--theme-color-primary);
+    position: absolute;
+    bottom: 2.21rem; /* 31px / 14 */
+    left: 50%;
+    transform: translateX(-50%);
+    border-radius: 1.43rem; /* 20px / 14 */
+    z-index: 2;
+}
+
+.hands_box .second {
+    width: 30rem; /* 420px / 14 */
+    height: 34.29rem; /* 480px / 14 */
+}
+
+.hands_box .second::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 1.14rem; /* 16px / 14 */
+    height: 1.14rem; /* 16px / 14 */
+    background: #F7410A;
+    border-radius: 1.79rem; /* 25px / 14 */
+}
+
+.hands_box .second::after {
+    content: '';
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 0.57rem; /* 8px / 14 */
+    height: 0.57rem; /* 8px / 14 */
+    background: var(--theme-color-primary);
+    border-radius: 1.79rem; /* 25px / 14 */
+}
+
+.hands_box .second i {
+    width: 0.29rem; /* 4px / 14 */
+    /* half of parent height*/
+    height: calc(15rem + 3.57rem); /* 210px + 50px / 14 */
+    background: #F7410A;
+    position: relative;
+    border-radius: 0.14rem; /* 2px / 14 */
+}
+
+.hands_box .second i::before {
+    content: '';
+    width: 0.29rem; /* 4px / 14 */
+    height: 3.21rem; /* 45px / 14 */
+    background: #F7410A;
+    position: absolute;
+    bottom: -2.93rem; /* 41px / 14 */
+    left: 50%;
+    transform: translateX(-50%);
+    border-radius: 0.36rem; /* 5px / 14 */
+}
+
+
+/*---------Clock End-----------*/
+
+
+/* Media Query */
+
+@media (orientation: portrait) {
+    .main-container {
+        flex-direction: column;
+    }
+    .primary-card {
+        width: 100%;
+        height: 50%;
+    }
+    .secondary-container {
+        width: 100%;
+        height: 50%;
+    }
+}
+
+/* 480px 720px Clock Container need to fix */
+/*TODO: Fix the Clock Circle issue : When Scale to .5, circle change to oval*/
+
+
+  @media screen and (width >=480px) and (orientation: portrait) {
+
+    .main-container {
+        gap: 1rem;
+        padding: 1rem;
+    }
+
+    .primary-card {
+        border-radius: 2rem;
+    }
+
```

Comment:
Common CSS should preferably be moved to a different CSS file, so that it can be reused directly.